src/components/views/messages/EncryptionEvent.tsx

```diff
@@ -40,21 +44,44 @@ const EncryptionEvent = forwardRef<HTMLDivElement, IProps>(({ mxEvent, timestamp
 
     const prevContent = mxEvent.getPrevContent() as IRoomEncryption;
     const content = mxEvent.getContent<IRoomEncryption>();
+    const card = useContext(CardContext);
 
     // if no change happened then skip rendering this, a shallow check is enough as all known fields are top-level.
     if (!objectHasDiff(prevContent, content)) return null; // nop
 
     if (content.algorithm === ALGORITHM && isRoomEncrypted) {
-        let subtitle: string;
+        let subtitle: React.ReactNode | string;
         const dmPartner = DMRoomMap.shared().getUserIdForRoomId(roomId);
         const room = cli?.getRoom(roomId);
         if (prevContent.algorithm === ALGORITHM) {
             subtitle = _t("Some encryption parameters have been changed.");
         } else if (dmPartner) {
-            const displayName = room?.getMember(dmPartner)?.rawDisplayName || dmPartner;
+            const dmPartnerRoomMember = room?.getMember(dmPartner);
+            const displayName = dmPartnerRoomMember?.rawDisplayName || dmPartner;
+            const profileLinkOnClick = (): void => {
+                if (!dmPartnerRoomMember) {
+                    // We were unable to fetch membership info for the other user in this DM. Do nothing.
+                    return;
+                }
+
+                // Display the other user's profile information.
+                dis.dispatch({
+                    action: Action.ViewUser,
+                    member: dmPartnerRoomMember,
+                    push: card.isCard,
+                });
+            };
+
             subtitle = _t(
-                "Messages here are end-to-end encrypted. Verify %(displayName)s in their profile - tap on their profile picture.",
+                "Messages here are end-to-end encrypted. Verify %(displayName)s in <a>their profile</a>.",
```

Comment:
This doesn't comply with translation key rules https://github.com/vector-im/element-web/blob/develop/docs/translating-dev.md

Reply:
Err, are you talking about the `%(displayName)s` bit?

Reply:
Unfortunately not, we're moving away from English-as-i18n-key to logical keys, you need to follow [key naming rules](https://github.com/vector-im/element-web/blob/develop/docs/translating-dev.md#key-naming-rules) and follow `Adding a new string` as it doesn't yet have a translation key (still uses English)